### PR TITLE
Bump go version to 1.18.1

### PR DESCRIPTION
**Pull request by ray-milkey**

### CODE REVIEW

**Comment by amolonf** [comment]
retest this please

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/onosproject/onos-config
 
-go 1.16
+go 1.18
```
Comment (adibrastegarnia) [inline go.mod:3]: FYI We need 1.19

Comment (amolonf) [reply go.mod:3]: Yes, I am trying to make it happen.

**Comment by amolonf** [comment]
retest this please

**Comment by amolonf** [comment]
retest this please

File: go.mod (hunk 1 above)
Comment (SeanCondon) [reply go.mod:3]: hi @amolonf - thanks for starting this
In the Jenkins console log I'm seeing
```
15:32:52 Building remotely on [docker-go1_16-009qys628y2ox on docker](https://jenkins.aetherproject.org/computer/docker-go1_16-009qys628y2ox) (docker-go1_16) in workspace /home/jenkins/workspace/verify_onos-config_make-test
```
good to know you're on it

**Comment by amolonf** [comment]
retest this please

File: go.mod (hunk 1 above)
Comment (amolonf) [reply go.mod:3]: I have update the go lang 1.19 but it does not help 
https://jenkins.aetherproject.org/job/verify_onos-config_make-test/894/ 
If you have access then you can check that the execution environment contains go lang 1.19

Comment (SeanCondon) [reply go.mod:3]: actually I'd recommend doing this on a simpler repo - like [onosproject/onos-api](https://github.com/onosproject/onos-api) - I'm not sure what dependencies are going on here

**Pull request closed without merging**